app/models/ability.rb

```diff
@@ -0,0 +1,34 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the user here. For example:
+    #
+    return unless user.present?
+
+    # user ||= User.new # Guest user (not logged in)
+    if user.admin?
+      can :manage, :all
+    else
+      can :read, Studio
+      can %i[create update destroy read], Reservation, user:
+    end
+    # The first argument to `can` is the action you are giving the user
+    # permission to do.
+    # If you pass :manage it will apply to every action. Other common actions
+    # here are :read, :create, :update and :destroy.
+    #
+    # The second argument is the resource the user can perform the action on.
+    # If you pass :all it will apply to every resource. Otherwise pass a Ruby
+    # class of the resource.
+    #
+    # The third argument is an optional hash of conditions to further filter the
+    # objects.
+    # For example, here the user can only update published articles.
+    #
+    #   can :update, Article, published: true
+    #
```

Comment:
- [ ] I'd argue that the link to the wiki is useful so we can leave it there 😁 But how about the other comments, are they necessary? 🕵🏽 Kindly, provide your opinion about this.